docs/basic_analysis.rst

```diff
@@ -76,6 +78,61 @@ And if the spectrum contains a continuum, then it should be subtracted first:
     >>> continuum = Spectrum1D(spectral_axis=spec.spectral_axis, flux=continuum_flux) #doctest:+SKIP
     >>> c = centroid(spec-continuum, region=None) #doctest:+SKIP
 
+Width
+-----
+
+There are several width statistics that are provided by the
+`~specutils.analysis` submodule.
+
+The `~gaussian_sigma_width` function estimates the width of the spectrum by
+computing an approximation of the standard deviation.
+
+The `~gaussian_fwhm` function estimates the width of the spectrum at half max,
+again by computing an approximation of the standard deviation.
+
+Both of these functions assume that spectrum is approximately gaussian.
+
+The function `~fwhm` provides an estimate of the full width of the spectrum at
+half max that does not assume the spectrum is gaussian. It locates the maximum,
+and then locates the value closest to half of the maximum on either side, and
+measures the distance between them.
+
+Consider the following noisy gaussian spectrum as an example:
+
+.. plot::
+   :include-source: true
+   :context:
+
+   >>> import numpy as np
+   >>> from astropy import units as u
+   >>> from astropy.modeling import models
+   >>> from specutils import Spectrum1D
+   >>> from specutils.analysis import gaussian_sigma_width
+   >>> np.random.seed(0)
+
+   >>> spectral_axis = np.linspace(0., 10., 200) * u.GHz
+   >>> spectral_model = models.Gaussian1D(amplitude=3*u.Jy, mean=5*u.GHz, stddev=0.8*u.GHz)
+   >>> flux = spectral_model(spectral_axis)
+   >>> # Add noise
+   >>> flux += np.random.normal(0., 0.2, spectral_axis.shape) * u.Jy
+   >>> noisy_gaussian = Spectrum1D(spectral_axis=spectral_axis, flux=flux)
+
+   >>> import matplotlib.pyplot as plt #doctest:+SKIP
+   >>> plt.plot(noisy_gaussian.spectral_axis, noisy_gaussian.flux) #doctest:+SKIP
+
+Each of the width analysis functions are applied to this spectrum below:
+
+.. code-block:: python
+
+   >>> from specutils.analysis import gaussian_sigma_width, gaussian_fwhm, fwhm
+   >>> gaussian_sigma_width(noisy_gaussian)
+   <Quantity 1.59661941 GHz>
```

Comment:
Travis is currently failing here.  You probably want a ``  # doctest: +FLOAT_CMP`` regardless (for other architectures, etc), but I think the actual *value* is also different? (should be more like 0.8, right?)

Reply:
Yes, this was an oversight.